vnclauncher.py

```diff
@@ -204,9 +204,8 @@ def _configure_vte(self):
         else:
             bg_color = '#FFFFFF'
             conf.set('terminal', 'bg_color', bg_color)
-        self.set_colors(Gdk.RGBA.parse(fg_color),
-                        Gdk.RGBA.parse(bg_color),
-                        [])
+        self.set_colors(Gdk.RGBA(*Gdk.color_parse(fg_color).to_floats()),
```

Comment:
This could be better, you can look at this [implementation](https://github.com/sugarlabs/bichos-activity/blob/f2c5e126c7694ab664d1f5804523c65b7c6b5845/Widgets.py) and decide how you want to proceed.